pandas/_libs/window/aggregations.pyx

```diff
@@ -1511,6 +1511,7 @@ def ewma(const float64_t[:] vals, const int64_t[:] start, const int64_t[:] end,
         s = start[j]
         e = end[j]
         sub_vals = vals[s:e]
+        sub_deltas = deltas[s:e - 1]
```

Comment:
why is this not `s:e`? (like sub_vals)

Reply:
Deltas are the (scaled) np.diff of the times vector. so ``len(deltas) = len(times) - 1 = len(vals) - 1``.

Reply:
kk can you add this comment on the shape of sub_vals vs sub_deltas for future readers